plsync/mlabconfig.py

```diff
@@ -505,14 +505,26 @@ def export_mlab_site_stats(output, sites):
                 'longitude': None
             }
         metro = [name, name[:-2]]
-        sitestats.append({
-            'site': name,
-            'metro': metro,
-            'city': location['city'],
-            'country': location['country'],
-            'latitude': location['latitude'],
-            'longitude': location['longitude']
-        })
+
+        if 'roundrobin' in site.keys():
```

Comment:
Should be 
```python
if site.keys().get('roundrobin', False):
```
That way we won't accidentally turn on Round Robin when passing in `roundrobin=False`